### DISCUSSION TITLE
Importing MRI scan into AMBF

### DISCUSSION THREAD

**Discussion by sunshineinsandiego** [WPI-AIM/ambf#212] [Q&A]
Hi - In the original [paper](https://arxiv.org/pdf/2302.13878.pdf) on AMBF, there's a brief section about the possibility of importing CT and MRI scans into AMBF. A few questions about this:

(1) Is there a recommended pipeline regarding how to do this?
(2) If 1. exists, does the above pipeline include tissue segmentation and assigning different physical properties to the tissues?
(3) If not 2., how should we go about assigning different dynamics and properties to different types of tissues as well as the links between them?

Thanks for any directions to resources that can help!

**Comment by adnanmunawar** [comment]
Hello,

Currently, the [ambf_slicer_plugin](https://github.com/adnanmunawar/ambf_util_slicer_plugin) is the simplest way to generate ADF files from volumetric data. These files, just like any other file, are loaded into AMBF. The volume can be segmented into different colors, all belonging to a single volume. You can also load multiple volumes. The interaction has to be implemented in a plugin and there you can query the color property and implement the desired interaction. For example, this set of plugins, implements, among other things, a voxel hardness property (https://github.com/htp2/continuum-manip-volumetric-drilling-plugin).
